### ISSUE TITLE
Barber Shop: attempt to index local 'f' (a nil value)

### ISSUE THREAD

**Issue by cloch** [Haleth/Aurora#51]
When opening the barber shop, I get a lua error message and the frame appears cut off. This issue also appears with all other addons turned off. 

**Screenshot:**
![barber shop](https://cloud.githubusercontent.com/assets/1244288/17774191/aa1ec2d4-6552-11e6-9ce2-72b717beca63.jpg)

**Error Message:**
```
2x Aurora\aurora-7.0.2.lua:401: attempt to index local 'f' (a nil value)
Aurora\aurora-7.0.2.lua:401: in function `ReskinArrow'
...rfaceAuroraBlizzard_BarbershopUI-1.0.lua:20: in function `addonModule'
Aurora\aurora-7.0.2.lua:808: in function <Aurora\aurora.lua:704>
[C]: in function `LoadAddOn'
FrameXML\UIParent.lua:391: in function `UIParentLoadAddOn'
FrameXML\UIParent.lua:474: in function `BarberShopFrame_LoadUI'
FrameXML\UIParent.lua:1484: in function <FrameXML\UIParent.lua:891>

Locals:
f = nil
direction = "left"
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'f' (a nil value)"
F = <table> {
 CreateBG = <function> defined @Aurora\aurora.lua:97
 colourArrow = <function> defined @Aurora\aurora.lua:258
 ReskinPortraitFrame = <function> defined @Aurora\aurora.lua:544
 Reskin = <function> defined @Aurora\aurora.lua:146
 ReskinDropDown = <function> defined @Aurora\aurora.lua:271
 dummy = <function> defined @Aurora\aurora.lua:84
 ReskinNavBar = <function> defined @Aurora\aurora.lua:631
 ReskinIcon = <function> defined @Aurora\aurora.lua:680
 ReskinArrow = <function> defined @Aurora\aurora.lua:400
 AddPlugin = <function> defined @Aurora\aurora.lua:692
 ReskinGarrisonPortrait = <function> defined @Aurora\aurora.lua:652
 ReskinFilterButton = <function> defined @Aurora\aurora.lua:612
 colourExpandOrCollapse = <function> defined @Aurora\aurora.lua:494
 CreateBDFrame = <function> defined @Aurora\aurora.lua:576
 clearExpandOrCollapse = <function> defined @Aurora\aurora.lua:501
 clearArrow = <function> defined @Aurora\aurora.lua:264
 ReskinTab = <function> defined @Aurora\aurora.lua:168
 SetBD = <function> defined @Aurora\aurora.lua:531
 ReskinInput = <function> defined @Aurora\aurora.lua:375
 ReskinClose = <function> defined @Aurora\aurora.lua:329
 ReskinColourSwatch = <function> defined @Aurora\aurora.lua:596
 ReskinRadio = <function> defined @Aurora\aurora.lua:451
 CreateBD = <function> defined @Aurora\aurora.lua:86
 ReskinExpandOrCollapse = <function> defined @Aurora\aurora.lua:509
 ReskinScroll = <function> defined @Aurora\aurora.lua:194
 CreateGradient = <function> defined @Aurora\aurora.lua:114
 ReskinSlider = <function> defined @Aurora\aurora.lua:476
 ReskinCheck = <function> defined @Aurora\aurora.lua:419
}
C = <table> {
 defaults = <table> {
 }
 classcolours = <table> {
 }
 media = <table> {
 }
 frames = <table> {
 }
 themes = <table> {
 }
 r = 1
 g = 0.22
 b = 0.52
 isBetaClient = true
 TOC = 70000
}
colourArrow = <function> defined @Aurora\aurora.lua:258
clearArrow = <function> defined @Aurora\aurora.lua:264

```

**Comment by Gethe** [comment]
Fixed in commit 5addb67

**Comment by cloch** [comment]
Argh, should have checked git version first before opening an issue. 

Thanks for your fast response and sorry for wasting your time,
